web/src/components/Icons.js

```diff
@@ -15,6 +16,7 @@ export {
   faBell,
   faChevronDown,
   faChevronUp,
+  faCode,
```

Comment:
this is needed for `DataInspector` component (forgot to add this to last PR) 🤦‍♂